devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -103,6 +104,36 @@
     @Parameter(defaultValue = "${debug}")
     private String debug;
 
+    /**
+     * Whether or not the JVM launch, in debug mode, should be suspended. This parameter is only
+     * relevant when the JVM is launched in {@link #debug debug mode}. This parameter supports the
+     * following values (all the allowed values are case insensitive):
+     * <table>
+     * <th>
+     * <td>Value</td>
+     * <td>Effect</td>
+     * </th>
+     * <tr>
+     * <td>true</td>
```

Comment:
Shouldn't we just have `true or y` and `false or n` and thus only 2 lines? I think it would be a bit better.

Reply:
Done. I've updated this PR with this change.